Dockerfile.dev

```diff
@@ -4,18 +4,16 @@ FROM python:3.6.6-alpine3.6
 LABEL maintainer="Senan Kelly <senan@senan.xyz>"
 RUN apk add --no-cache libev build-base libffi-dev
 COPY ./requirements-docker.txt /
-RUN --mount=type=cache,target=/root/.cache/pip \
```

Comment:
any reason to get rid of the `--mount=type=cache,target=/root/.cache/pip`? when running with `COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1` this seems to help with faster builds while development (though maybe not I haven't done comparisons)